grpc-sys/grpc_wrap.cc

```diff
@@ -673,14 +680,17 @@ grpcwrap_call_send_close_from_client(grpc_call* call, void* tag) {
 GPR_EXPORT grpc_call_error GPR_CALLTYPE grpcwrap_call_send_status_from_server(
     grpc_call* call, grpcwrap_batch_context* ctx, grpc_status_code status_code,
     const char* status_details, size_t status_details_len,
-    grpc_metadata_array* trailing_metadata, int32_t send_empty_initial_metadata,
-    grpc_slice* optional_send_buffer, uint32_t write_flags, void* tag) {
+    grpc_metadata_array* initial_metadata, grpc_metadata_array* trailing_metadata,
+    int32_t send_empty_initial_metadata, grpc_slice* optional_send_buffer,
```

Comment:
I suggest to remove `send_empty_initial_metadata`. Caller can pass an empty initial metadata if necessary.

Reply:
Good idea, I'll do that.